### PR TITLE
[#562] adding multiple reviewers to topic

### PR DESCRIPTION
**Pull request by tangrammer**

### CODE REVIEW

File: backend/src/gpml/handler/review.clj [hunk 1]
```diff
@@ -37,29 +37,36 @@
                 {:topic-type topic-type :topic-id topic-id})]
     (resp/response review)))
 
-(defn new-review [db mailjet-config topic-type topic-id reviewer-id assigned-by]
+(defn create-review-with-multiple-reviewers [topic-type topic-id assigned-by conn reviewer-id]
```
Comment (tangrammer) [inline backend/src/gpml/handler/review.clj:40]: the function name doesn't relate with the logic inside nor the args received ... the logic inside only create a review with a reviewer

File: backend/project.clj [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (tangrammer) [inline backend/project.clj:1]: can you please remove this change?

File: docker-compose.override.yml [hunk 3]
```diff
@@ -18,7 +18,7 @@ services:
       - db
   backend:
     image: akvo/akvo-clojure-lein:20210124.114043.4437caf
-    command: ["lein", "repl", ":headless"]
+    command: ["tail", "-f", "/dev/null"]
```
Comment (tangrammer) [inline docker-compose.override.yml:21]: IMO these settings are only useful in your env, so please remove them from this PR

**Comment by tangrammer** [comment]
@Pawan05 can you rebase this branch with main?

File: backend/project.clj (hunk 2 above)
Comment (Pawan05) [reply backend/project.clj]: Addressed [here|https://github.com/akvo/unep-gpml/commit/2843790c7ed570ec976ab5e3f9e92eb65779b6df#diff-55baed1d837e794d64536d1f19c52102be6963761097f7577b82ad0891d6827cL10]

File: docker-compose.override.yml (hunk 3 above)
Comment (Pawan05) [reply docker-compose.override.yml]: Removed and replaced with newer values https://github.com/akvo/unep-gpml/commit/2843790c7ed570ec976ab5e3f9e92eb65779b6df#diff-2286cdeca85adb587a9640aa34352ba7c2fb6e49c0129dab05892ff67d0d4c58R21

**Pull request closed without merging**